python-packages/contract_wrappers/src/zero_ex/contract_wrappers/__init__.py

```diff
@@ -131,39 +133,215 @@
 >>> (taker_allowance) // 10 ** 18
 100
 
-To give our accounts some initial WETH balances, we'll need
-to "wrap" some ETH to get WETH. WETH contains two extra methods,
-not included in the ERC20 token standard, so we will grab
-the ABI for the WETH Token contract and call the deposit method
-to wrap our ETH. Here is how we do so.
+To give our accounts some initial WETH balance, we'll need
+to *wrap* some ETH to get WETH. The WETH token contract
+contains two extra methods, not included in the ERC20 token
+standard, so we will grab the ABI for the WETH Token contract
+and call the deposit method to wrap our ETH. Here is how we do so.
 
 >>> from zero_ex.contract_artifacts import abi_by_name
->>> # Converting 1 ETH to base unit wei
->>> deposit_amount = 1 * 10 ** 18
+>>> # Converting 0.5 ETH to base unit wei
+>>> deposit_amount = int(0.5 * 10 ** 18)
 
 >>> # Let's have our maker wrap 1 ETH for 1 WETH
 >>> tx = erc20_wrapper.execute_method(
 ... address=weth_address,
 ... abi=abi_by_name("WETH9"),
 ... method="deposit",
 ... tx_opts = {"from_": maker, "value": deposit_amount})
->>> # Checking our maker balance
+>>> # Checking our maker's WETH balance
 >>> maker_balance = erc20_wrapper.balance_of(
 ...     token_address=weth_address, owner_address=maker)
->>> (maker_balance) // 10 ** 18  # doctest: +SKIP
-1
+>>> (maker_balance) / 10 ** 18  # doctest: +SKIP
+0.5
 
->>> # Let's have our taker wrap 1 ETH as well
+>>> # Let's have our taker wrap 0.5 ETH as well
 >>> tx = erc20_wrapper.execute_method(
 ... address=weth_address,
 ... abi=abi_by_name("WETH9"),
 ... method="deposit",
 ... tx_opts = {"from_": taker, "value": deposit_amount})
->>> # Checking our maker balance
+>>> # Checking our taker's WETH balance
 >>> taker_balance = erc20_wrapper.balance_of(
 ...     token_address=weth_address, owner_address=taker)
->>> (taker_balance) // 10 ** 18  # doctest: +SKIP
-1
+>>> (taker_balance) / 10 ** 18  # doctest: +SKIP
```

Comment:
Why skip?  This will always be the same value, right?

Reply:
skipping to avoid issues in case other tests change the balance